docs/FAQ.md

```diff
@@ -296,13 +296,13 @@ It's up to you:
 
 **Pros**
 
-- it's the only way to get truly reproducible builds, as it guards against upstream renames and deletes
-- you don't need an extra `dep ensure` step (to fetch dependencies) on fresh clones to build your repo
+- It's the only way to get truly reproducible builds, as it guards against upstream renames and deletes.
+- You don't need an extra `dep ensure` step to sync `vendor/` with Gopkg.lock after most operations, such as cloning, getting latest, merging, etc.
 
 **Cons**
 
-- your repo will be bigger, potentially a lot bigger
-- PR diffs are more annoying
+- Your repo will be bigger, potentially a lot bigger. Pruning can help minimize this problem.
```

Comment:
Minor nit:

Let's remove "Pruning can help minimize this problem." when prune is merged into ensure. Since it will feel like an additional step is required to have that benefit.